src/handlers/payout/action.ts

```diff
@@ -220,9 +221,17 @@ export const incentivesCalculation = async (): Promise<IncentivesCalculationResu
     throw new Error(errMsg);
   }
 
+  const permitComments = comments.filter((content) => content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot);
+
+  if (permitComments.length > 0) {
+    logger.info(`skip to generate a permit url because it has been already posted`);
+    throw new Error(`skip to generate a permit url because it has been already posted`);
```

Comment:
We are using a new error function, something like `ErrorDiff` @Keyrxng do the errors bubble up from the handlers into `ErrorDiff`?

Reply:
> ErrorDiff

The error is thrown and caught here: https://github.com/devpanther/bounty-bot/blob/544b61175dbfcf0fb5fb1d8c7b2ab45798b6940c/src/handlers/comment/handlers/index.ts#L110

Which uses the `ErrorDiff`